source/index.html.md

```diff
@@ -233,16 +252,43 @@ After an user creates an account with AnchorX, the service will
 automatically provision a Stellar account and authorize the account to
 hold the anchor's asset.
 
-When a Stellar accounts decides to trust a given asset, they are
-creating a trustline between the account and the asset. Such operation
-has to be stored in the ledger. The code in the right shows a transaction creating a trustline between an `account` and an `asset`.
+[NOTE: Confussing. Too much new information: "ledger/trustline/trustor" and the code
```

Comment:
@abuiles I found this part the most confusing on my review. I know there are a lot of diffs, but I'd focus on this section as the most important from this PR.